### ISSUE TITLE
Skill Checks Broken: Uncaught TypeError: Cannot read property 'altKey' of undefined

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#137]
In GitLab by @joshbas0300 on Nov 18, 2020, 06:53

When only Midi QoL is active with default settings and console printing enabled, when you try and perform a skill check this error gets printed to the console:

```
patching.js:107 Uncaught TypeError: Cannot read property 'altKey' of undefined
    at Actor5e.doRollSkill (patching.js:107)
    at ActorSheet5eCharacter._onRollSkillCheck (base.js:743)
    at HTMLHeadingElement.dispatch (jquery.min.js:2)
    at HTMLHeadingElement.v.handle (jquery.min.js:2)
doRollSkill @ patching.js:107
_onRollSkillCheck @ base.js:743
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```

**Comment by tposney** [comment]
In GitLab by @liquidmetal17 on Nov 18, 2020, 07:30

I was just running into this problem.  If you go into the Midi-Qol Configuration and check both "Enable Speed Item Rolls" and "Enable Speed Ability(save/check/skill) Rolls, skill checks will roll without error.

**Comment by tposney** [comment]
In GitLab by @stefanu9115 on Nov 18, 2020, 07:30


same mistake

**Comment by tposney** [comment]
Thanks for the report. A typo in the guard test. Fix will go out in 0.3.27
